src/commands/dev.py

```diff
@@ -0,0 +1,126 @@
+"""Developer Commands"""
+
+import os
+
+from discord import Activity, ActivityType, Game
+from discord.ext.commands import Bot, Cog, Context, command, is_owner, parameter
+
+from utils.cog import get_cog_path, reload_modules
+from utils.embed import create_error_embed
+from utils.logging import get_logger
+
+LOGGER = get_logger(os.path.basename(__file__))
+
+
+class Dev(Cog):
+    """Cog defining developer commands.
+    These commands are hidden and will only be available to users with elevated permissions
+    """
+
+    def __init__(self: "Dev", bot: Bot) -> None:
+        """Init Cog
+
+        Args:
+            bot (`Bot`): Discord Bot object
+        """
+        self.bot = bot
+
+    @command(name="load", hidden=True)
+    @is_owner()
+    async def load_cog(self: "Dev", ctx: Context, *, cog: str) -> None:
```

Comment:
[For all commands in this cog] Add Discord `parameter()`. Also get rid of * param since it isn't needed

Reply:
also fix docstring wordings, a little weird